vmlinux/riscv/vmlinux.h

```diff
@@ -0,0 +1 @@
+./vmlinux_519.h
```

Comment:
oh, also seems like other architectures have vmlinux.h generated from 6.1 or 6.2 kernels, can you please regenerate from the 6.2 kernel as well?

Reply:
Updated to 6.2 kernel, thanks.